package.json

```diff
@@ -60,7 +60,7 @@
     "istanbul-instrumenter-loader": "^3.0.0",
     "karma": "^4.0.0",
     "karma-babel-preprocessor": "^6.0.1",
-    "karma-browserstack-launcher": "1.4.0",
+    "karma-browserstack-launcher": "1.5.1",
```

Comment:
I deliberately change it to `1.4.0` in this PR, https://github.com/prebid/Prebid.js/pull/4840. We need to do this because in version 1.5.1, karma-browserstack-launcher times out. Here's the link to the open issue: https://github.com/karma-runner/karma-browserstack-launcher/issues/152

With, `v1.4.0`, i've seen no such problem so far (tests timing out).